packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -456,7 +456,7 @@ export abstract class SnapshotCredentials {
 /**
  * Properties common to single-user and multi-user rotation options.
  */
-interface CommonRotationUserOptions {
+export interface CommonRotationUserOptions {
```

Comment:
Unfortunately not possible to have a public interface that extends an interface from a non exported file with `jsii`...


```
error JSII9002: Unable to resolve type "@aws-cdk/aws-rds.CommonRotationUserOptions". It may be @internal or not exported from the module's entry point (as configured in "package.json" as "main").
```

Reply:
That's weird, because this interface used to be not exported, but JSII is gonna JSII...😉. Let's leave it as-is.